plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -1130,6 +1130,7 @@ $default-line-height: 18px;
 	}
 	.toolbar-top {
 		.button,
+		.select,
```

Comment:
Instead of using a class selector here (`.select`), could you just use the `select` element selector?

Or, if you need more specificity, why not just use the `attribute_taxonomy` class?